config/routes.rb

```diff
@@ -65,14 +65,8 @@ def external_redirect(path_prefix, target)
     resources :fatality_notices, path: "fatalities", only: [:show]
     get "/ministers(.:locale)", as: "ministerial_roles", to: "ministerial_roles#index", constraints: { locale: valid_locales_regex }
     get "/uploads/system/uploads/attachment_data/file/:id/*file.:extension/preview" => "csv_preview#show", as: :csv_preview
-
-    get "/consultations", as: "consultations", to: "consultations#index" # used only for a redirect
-    get "/statistics/announcements", as: "statistics_announcements", to: "statistics_announcements#index" # used only for a redirect
     # End of public facing routes still rendered by Whitehall
 
-    # Routes no longer rendered by Whitehall, but retained to maintain the route helpers
-    get "/statistics(.:locale)", as: "statistics", to: "statistics#index", constraints: { locale: valid_locales_regex }
```

Comment:
Am I missing something here, or do we actually still have to do the work to do the statistics redirect? This wasn't in your PR (that was /statistics/announcements and /consultations, and the [content item](https://www.gov.uk/api/content/government/statistics) still has a rendering app of whitehall

Reply:
Ah, I have forgotten something. But now what you'd think. Somehow the `statistics` redirect has been hacked straight into router without a content item, so the query params for this haven't been handled for the last two years, and given that as far as me and Bruce could tell this redirect was for non-english finders, which we no longer support, we decided it wasn't very important. What I did forget was to add something into short-url-manager for this (and likely special-routes-publisher) so there is at least a content item. I can do that shortly.

Reply:
Cool, let me know when done and I can have another look